cql/pom.xml

```diff
@@ -141,10 +147,10 @@
                             !org.slf4j.spi,
                             !sun.misc,
                             !org.eclipse.jetty.security,
```

Comment:
a bit tangential to this PR, but in general, would it make sense to import only specific required packages and then have a wildcard `!*` at the end of the imports section?

Reply:
I _think_ that by default when specifying `<Import-Package>`, it won't include `*`, so negating here might be unnecessary, but 🤷

Reply:
I don't think we need the negating. We should probably just import what we use.